docs/user/device-observability.md

```diff
@@ -272,7 +272,8 @@ One way to extract the CA:
 
 ```yaml
```

Comment:
Nit: the script type shhould be `bash`
Also, for consistency with the other commands, the command should use ./bin/flightctl...